nestkernel/recording_device.h

```diff
@@ -42,6 +42,62 @@
 namespace nest
 {
 
+/* BeginUserDocs: NOINDEX
+
+Recording time window
++++++++++++++++++++++
+
+The time span during which the recorder actively records can be
+specified using the properties ``start`` and ``stop``. These define
+the activation period of the device in ms. An additional property
```

Comment:
If find `activation period` to be a bit ambiguous in this case since to my knowledge in a Neuroscientific context it's usually related to neural activation. Maybe `recording period` is better?